app/src/main/java/org/literacyapp/authentication/CameraViewActivity.java

```diff
@@ -0,0 +1,156 @@
+package org.literacyapp.authentication;
+
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.SurfaceView;
+
+import org.literacyapp.LiteracyApplication;
+import org.literacyapp.R;
+import org.literacyapp.dao.StudentImage;
+import org.literacyapp.dao.StudentImageCollectionEventDao;
+import org.literacyapp.util.DeviceInfoHelper;
+import org.literacyapp.util.MultimediaHelper;
+
+import org.opencv.android.CameraBridgeViewBase;
+import org.opencv.android.JavaCameraView;
+import org.opencv.android.OpenCVLoader;
+import org.opencv.core.Core;
+import org.opencv.core.Mat;
+import org.opencv.core.Rect;
+
+
+import java.io.File;
+import java.util.Date;
+
+
+import ch.zhaw.facerecognitionlibrary.Helpers.FileHelper;
+import ch.zhaw.facerecognitionlibrary.Helpers.MatName;
+import ch.zhaw.facerecognitionlibrary.Helpers.MatOperation;
+import ch.zhaw.facerecognitionlibrary.PreProcessor.PreProcessorFactory;
+
+public class CameraViewActivity extends AppCompatActivity implements CameraBridgeViewBase.CvCameraViewListener2 {
+    private JavaCameraView preview;
+    private PreProcessorFactory ppF;
+    private Boolean diagnoseMode;
+    private long timerDiff;
+    private long lastTime;
+    private int numberOfPictures;
+    private int count;
+    private String deviceId;
+    private String collectionEventId;
+    private StudentImageCollectionEventDao studentImageCollectionEventDao;
+
+
+
+    static {
+        if (!OpenCVLoader.initDebug()) {
+            // Handle initialization error
+        }
+    }
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        diagnoseMode = true;
+        count = 0;
+
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_camera_view);
+
+        preview = (JavaCameraView) findViewById(R.id.CameraView);
+
+        preview.setCameraIndex(1);
+
+        preview.setVisibility(SurfaceView.VISIBLE);
+        preview.setCvCameraViewListener(this);
+
+        lastTime = new Date().getTime();
+        timerDiff = 100;
```

Comment:
Move these values to static final variables at the top of the file for easier configuration?